components/docusign/sources/new-folder/new-folder.js

```diff
@@ -0,0 +1,68 @@
+const common = require("../common.js");
+
+module.exports = {
+  ...common,
+  key: "docusign-new-folder",
+  name: "New Folder",
+  description: "Emits an event when a new folder is created",
+  version: "0.0.1",
+  dedupe: "unique",
+  methods: {
+    ...common.methods,
+    _getFolderIds() {
+      return this.db.get("folderIds");
+    },
+    _setFolderIds(folderIds) {
+      this.db.set("folderIds", folderIds);
+    },
+    processFolder(folderIds, folder, ts) {
+      if (this.isRelevant(folderIds, folder.folderId)) {
```

Comment:
You don't need to store the folder IDs and dedupe when using `unique`, Pipedream will do the work for you (https://pipedream.com/docs/components/quickstart/nodejs/sources/#only-emit-new-items-in-the-rss-feed).

Reply:
Updated